zerver/lib/message.py

```diff
@@ -407,6 +408,82 @@ def is_subscribed_helper() -> bool:
     return is_subscribed_helper()
 
 
+def event_recipient_ids_for_action_on_messages(
+    messages: list[Message],
+    *,
+    channel: Stream | None = None,
+    exclude_long_term_idle_users: bool = False,
```

Comment:
Should we always exclude `long_term_idle` users ?

Is there any special reason to do this only in the case of `delete_message..`

Reply:
I think the optimization was important for delete messages, so we did it there, but probably we should do it always.

Reply:
Made this change.